README.md

```diff
@@ -111,12 +111,18 @@ This command will retrieve the given package manager from the specified archive
 
 ## Environment Variables
 
-- `COREPACK_ENABLE_NETWORK` can be set to `0` to prevent Corepack from accessing the network (in which case you'll be responsible for hydrating the package manager versions that will be required for the projects you'll run, using `corepack hydrate`).
-
 - `COREPACK_DEFAULT_TO_LATEST` can be set to `0` in order to instruct Corepack
   not to lookup on the remote registry for the latest version of the selected
   package manager.
 
+- `COREPACK_ENABLE_NETWORK` can be set to `0` to prevent Corepack from accessing
+  the network (in which case you'll be responsible for hydrating the package
+  manager versions that will be required for the projects you'll run, using
+  `corepack hydrate`).
+
+- `COREPACK_ENABLE_STRICT` can be set to `0` to prevent Corepack from checking
```

Comment:
It's a _bit_ weird that the name is `COREPACK_ENABLE_STRICT` but the default is `true`. You only use this environment variable when you want to _disable_ strict checking.

Reply:
It follows the same pattern as `COREPACK_ENABLE_NETWORK`. Do you have another name suggestion?

Reply:
I prefer env vars and booleans alike are always "enable" regardless of default value so you don't have to think about double natives of disabling the "disable" variable.

Reply:
Separately tho, it's always weird and confusing when a boolean option has different behavior between "absent" and "false".

Reply:
I don't have another naming suggestion given that this follows an existing pattern. I'm not trying to block or delay this with my comment.